core/roboconf-core/src/main/java/net/roboconf/core/Constants.java

```diff
@@ -61,13 +61,22 @@
 	 */
 	String FILE_EXT_RULE = ".drl";
 
-
 	/**
 	 * The wild card symbol (can be used in a component to reference any variable for a given group).
 	 */
 	String WILDCARD = "*";
 
 	/**
+	 * The default name of a script.
+	 * */
+	String SCRIPT_NAME = "script";
+
+	/**
+	 * The default name of a script.
+	 * */
+	String EXEC_SCRIPT = "script-all";
```

Comment:
Roboconf always executes ONE script. This one contains "-script." in its name.
Users can provide several scripts. If so, one will be executed by Roboconf. Others will act as libraries for the script to be executed.